tasks/e2e.sh

```diff
@@ -135,7 +138,7 @@ npm install $cli_path
 
 # Install the app in a temporary location
 cd $temp_app_path
-create_react_app --scripts-version=$scripts_path test-app
+create_react_app --scripts-version=$scripts_path --template=$root_path/packages/react-scripts/fixtures/kitchensink test-app
```

Comment:
You mean that I should add the `e2e-kitchensink.sh` test in `.travis.yml` after the `e2e.sh` one, or that they should be diversified based for example on commit/pull request...?